src/controllers/csi/provisioner/agent.go

```diff
@@ -76,7 +76,7 @@ func newAgentImageUpdater( //nolint:revive // argument-limit doesn't apply to co
 		fs:            fs,
 		path:          path,
 		targetDir:     path.AgentConfigDir(tenantUUID),
-		targetVersion: dk.CodeModulesImageTag(),
```

Comment:
ImageTag my or may not be set, and in the `ImageInstaller`'s case, this is only used for logging, therefor its more helpful to have the image in the log, then potentially an empty string